src/plugins/woocommerce-malga-payments/woocommerce-malga-payments.php

```diff
@@ -5,6 +5,8 @@
  * Description: Take credit card payments on your store using Malga.
  * Author: Malga Team
  * Author URI: https://www.malga.io/
+ * License: GPLv2
+ * License URI: https://www.gnu.org/licenses/gpl-3.0.html
```

Comment:
Bota assim pra esses casos de estar explicitado na v2

```suggestion
 * License: GPLv2 or later
 * License URI: https://www.gnu.org/licenses/gpl-2.0.html
```

Reply:
opsss, mandou mt